db/model/generatortemplate.js

```diff
@@ -120,11 +129,7 @@ module.exports = function user(seq, dataTypes) {
         is: constants.versionRegex,
       },
     },
-    bulk: {
-      type: dataTypes.BOOLEAN,
```

Comment:
https://salesforce.github.io/refocus/docs/40-coredevelopment.html

the migration tool generates the file (named based on timestamp), you just give it some short descriptive name which gets appended to the filename it generates